### ISSUE TITLE
Error when running 2.3 on the web - "window.performance.now is not a function"

### ISSUE THREAD

**Issue by dannyhw** [software-mansion/react-native-reanimated#2729]
## Description

When I setup react-native-web with reanimated I get the following error 

```
JSReanimated.js:1 Uncaught TypeError: window.performance.now is not a function
    at Object.now (JSReanimated.js:1)
    at JSReanimated.getTimestamp (JSReanimated.js:1)
    at eval (JSReanimated.js:1)
```

This could be an issue in my webpack config but I've tried a lot of things to fix it and I couldn't seem to arrive at a solution. 

Also I ran this in the latest version of chrome so it's not because of an old browser

### Expected behavior

No error and animations work

### Actual behavior & steps to reproduce

My animated component doesn't work and I get the error.

## Snack or minimal code example

I have a reproduction repo where I have an example showing this problem

If you run the following it should clone the single branch and run the app
```
git clone --single-branch -b gesture-reanimated git@github.com:dannyhw/react-native-reproduction.git && \
  cd react-native-reproduction && \
  yarn && \
  yarn web
```

Then open the console in the browser and try dragging the hello world text around, nothing happens but the error is printed a lot of times.

you can see the code here in this PR
https://github.com/dannyhw/react-native-reproduction/pull/3

A workaround for this issue is to set window.performance.now to date.now in index.web.js

```
if (!window.performance.now) window.performance.now = Date.now;
```

## Package versions

- React Native: 0.66.3
- React Native Reanimated: ^2.3.0
- NodeJS: v14.18.1
- Xcode: 13
- Java & Gradle: openjdk version "1.8.0_292", Gradle 6.9

## Affected platforms

- [ ] Android
- [ ] iOS
- [x] Web

**Comment by tomekzaw** [comment]
Hello! This should be fixed with #2718 which has already landed on the main branch but has not been released yet. Please check if it resolves the bug and feel free to close the issue if it helps.

**Comment by dannyhw** [comment]
@tomekzaw awesome, I'll try it out thanks!

**Comment by dannyhw** [comment]
@tomekzaw whats the best way to test this change? is there a beta version with it in?

**Comment by tomekzaw** [comment]
@dannyhw The best way would be to manually apply the diff from #2718 in `node_modules/react-native-reanimated/src/reanimated2/core.ts`.

Alternatively, you can run `yarn add react-native-reanimated#main` to use the current version from our main branch in your project (since the web version of Reanimated has no native code).

**Comment by dannyhw** [comment]
Ok great thanks for the explanation 🙏

**Comment by dannyhw** [comment]
@tomekzaw I can confirm its resolved by the changes in #2718 so I'll close this issue. Thanks for pointing me in the right direction. It's relatively easy to apply a patch-package for this so I'll be using that for now.

**Comment by dannyhw** [comment]
```
diff --git a/node_modules/react-native-reanimated/lib/reanimated2/core.js b/node_modules/react-native-reanimated/lib/reanimated2/core.js
index 706a560..117505d 100644
--- a/node_modules/react-native-reanimated/lib/reanimated2/core.js
+++ b/node_modules/react-native-reanimated/lib/reanimated2/core.js
@@ -1,7 +1,7 @@
 /* global _WORKLET _getCurrentTime _frameTimestamp _eventTimestamp, _setGlobalConsole */
 import NativeReanimatedModule from './NativeReanimated';
 import { Platform } from 'react-native';
-import { nativeShouldBeMock, shouldBeUseWeb } from './PlatformChecker';
+import { nativeShouldBeMock, shouldBeUseWeb, isWeb } from './PlatformChecker';
 global.__reanimatedWorkletInit = function (worklet) {
     worklet.__worklet = true;
 };
@@ -263,8 +263,8 @@ if (!NativeReanimatedModule.useOnlyV1) {
     NativeReanimatedModule.installCoreFunctions(NativeReanimatedModule.native
         ? workletValueSetter
         : workletValueSetterJS);
-    const capturableConsole = console;
-    isConfigured() &&
+    if (!isWeb() && isConfigured()) {
+        const capturableConsole = console;
         runOnUI(() => {
             'worklet';
             const console = {
@@ -279,6 +279,7 @@ if (!NativeReanimatedModule.useOnlyV1) {
                 now: global._chronoNow,
             };
         })();
+    }
 }
 let featuresConfig = {
     enableLayoutAnimations: false,

```

Heres the patch I used in case anyone needs it
